core/src/main/java/com/linecorp/armeria/common/AbstractContextAwareExecutorService.java

```diff
@@ -0,0 +1,107 @@
+package com.linecorp.armeria.common;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+abstract class AbstractContextAwareExecutorService implements ExecutorService {
+    protected final ExecutorService executor;
```

Comment:
How about removing `protected`? It's a package-local class anyway.

Reply:
OK